front/src/components/Collaborator.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CreateProfile from './CreateProfile';
+import NotConnected from './NotConnected';
+import SearchComplianceOfficer from './SearchComplianceOfficer';
+
+export default function Collaborator(props) {
+  const user = props.user;
+  const updateUser = (userUpdated) => {
+    props.updateUser(userUpdated);
+  };
+
+  return (
+    <>
+      {!user ? (
+        <NotConnected />
+      ) : !user.accountHasProfile ? (
+        <CreateProfile
+          user={user}
+          updateUser={updateUser}
+        />
+      ) : (
+        <SearchComplianceOfficer user={user} />
+      )}
```

Comment:
I'm not a big fan of chained ternaries. Please use old good if/else that will be way more readable.

Reply:
I'm a ternary-_Slamist_ sorry :) 
only expressions are available in fact in the return of JSX component  statements are not permitted